### PR TITLE
Update playwright to 1.28.1

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update playwright to 1.28.1
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

**Comment by nesrineabdmouleh** [comment]
https://github.com/nesrineabdmouleh/PrestaShop/actions/runs/3565501692

File: .github/workflows/myTesting.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      PS_LANGUAGE: 'en'
+      PS_INSTALL_AUTO: 1
+      PS_DEV_MODE: 0
+      DB_SERVER: 'mysql'
+      DB_NAME: 'prestashop'
+      DB_PREFIX: 'tst_'
+      DB_USER: 'root'
+      DB_PASSWD: 'prestashop'
+
+    steps:
```
Comment (lartist) [inline .github/workflows/myTesting.yml]: Should we use a "Cancel Previous with all but latest" step in order to ease the CI?

**Review by lartist — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]
:tada:

**Review by mflasquin — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @nesrineabdmouleh

**Pull request merged**